packages/scss/src/components/switch/index.scss

```diff
@@ -38,7 +30,7 @@
 		@include hover;
 	}
 
-	&:is(:focus-within) {
+	&:is(:focus-visible) {
```

Comment:
Just to make sure this isn't a mistake as `:focus-within` and `:focus-visible` won't have the same effect.